help/destinations/catalog/mobile-engagement/line.md

```diff
@@ -0,0 +1,141 @@
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| `[!DNL Channel access token (long-lived)]` | Your [!DNL LINE Channel access token (long-lived)]. | `aaa2112XSMWqLXR7..........nyilFU=` |
+
+Refer to the [[!DNL LINE] documentation](https://developers.line.biz/en/docs/messaging-api/getting-started/) for guidance.
+
+## Supported identities {#supported-identities}
+
+[!DNL LINE] supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+| Target Identity | Description | Considerations |
+|---|---|---|
+|IDFA|Apple ID for Advertisers|Select the IDFA target identity when your source identity is an IDFA namespace.|
+|UserID| Identifier for the LINE contact.|Select the UserID target identity when your source identity is UserID. |
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | You are exporting all members of a segment (audience) with the identifiers (name, phone number, or others) used in the [!DNL LINE] destination. |
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations). |
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+>[!IMPORTANT]
+> 
+> To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]** search for [!DNL LINE]. Alternatively you can locate it under the **[!UICONTROL Mobile engagement]** category.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, select **[!UICONTROL Connect to destination]**.
+![Platform UI screenshot showing how to authenticate.](../../assets/catalog/mobile-engagement/line/authenticate-destination.png)
+
+Fill in the required fields below.
+* **[!UICONTROL Bearer token]**: Your [[!DNL LINE Channel access token (long-lived)] from the LINE developer console](#gather-credentials).
```

Comment:
Suggestion to change the link. We don't need to hyperlink the entire sentence: 

`Your [[!DNL LINE Channel access token (long-lived)]](#gather-credentials) from the LINE developer console.`